src/overlays/actors/ovl_En_Bb/z_en_bb.c

```diff
@@ -10,17 +10,90 @@
 
 #define THIS ((EnBb*)thisx)
 
+#define bombHopPhase enBb_26C
+#define trailIdx enBb_26C
+#define trailMaxAlpha enBb_26E
+#define moveAngleY enBb_26E
+#define flameTimer enBb_26E
+
```

Comment:
If you understand this actor well enough to keep documenting, maybe it's worth doing some more defines for `actionState`? If I'm reading this correctly, `actionState` is also used for different things in different actors (it's even a flag in one of them which confused me at first).